### PR TITLE
Add more logs to RDMA drivers

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
Just adding more logs and `RDMA:` prefixes where applicable in case something goes wrong and we need troubleshooting.

@rjschwei I appreciate your reviews on suse/rdma.py

### CODE REVIEW

File: azurelinuxagent/common/rdma.py [hunk 1]
```diff
@@ -82,11 +83,16 @@ def is_driver_loaded(self):
...
+            logger.info('RDMA: module loaded.')
             return True
+        logger.info('RDMA: module not loaded.')
 
     def reboot_system(self):
         """Reboot the system. This is required as the kernel module for
            the rdma driver cannot be unloaded with rmmod"""
-        logger.info('System reboot')
-        shellutil.run('shutdown -r now')
+        logger.info('Rebooting system.')
```
Comment (rjschwei) [inline azurelinuxagent/common/rdma.py]: Should probably add the "RDMA:" prefix to this message as well so we know the reboot was initiated from the RDMA code

File: azurelinuxagent/common/rdma.py [hunk 2]
```diff
@@ -82,11 +83,16 @@ def is_driver_loaded(self):
...
+        ret = shellutil.run('shutdown -r now')
+        if ret != 0:
+            logger.error('Failed to reboot the system')
```
Comment (rjschwei) [inline azurelinuxagent/common/rdma.py]: Dito "RDMA:" prefix is probably a good idea

**Comment by rjschwei** [comment]
Other than the potentially missing "RDMA:" prefixes in two messages this is IMHO good to go

**Comment by hglkrijger** [comment]
lgtm

**Pull request merged**